test/core/slice/slice_buffer_test.cc

```diff
@@ -104,11 +105,49 @@ void test_slice_buffer_move_first() {
   GPR_ASSERT(dst.length == dst_len);
 }
 
+static void populate_slice_buffer(grpc_slice_buffer* sb) {
+  grpc_slice slices[4];
```

Comment:
This function may not be very representative. 
1. The slice buffer uses a 8 element slice array to inline the slices and you will always hit that.
2. The slice itself inlines any string shorter than 24B
Thus you always hit the inline case for both.

Reply:
With my new changes, this PR should actually be covered in tcp_posix_test. I will read over it to see if I need to make changes to test this more